plugins/example/example.coffee

```diff
@@ -0,0 +1,22 @@
+(() ->
+  class HelloWorldPlugin
+    @metadata =
+      name: "Hello World"
+      author: "Zachary Vance"
+      description: "Prints 'Hello World' when the plugin is loaded"
+      version: 1
+      stores_data: false
+      #data_version: 0
+      requires: []
+
+    constructor: (@api) ->
+      do @enable
+    
+    enable: () ->
+      console.log "Hello world"
+
+    # exports
+    module?.exports = HelloWorldPlugin
+    if window?
+      (window.PluginList ?= []).push HelloWorldPlugin
```

Comment:
i think rather than giving the plugins access to a list like this, we should just give them access to the registerPlugin function and have them go

```
registerPlugin(me)
```